how-to-use-azureml/machine-learning-pipelines/intro-to-pipelines/aml-pipelines-schedule-with-pipeline-parameter-built-in-functions.ipynb

```diff
@@ -0,0 +1,296 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Define pipeline parameters"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azureml.data.datapath import DataPath\n",
+    "\n",
+    "pipeline_parameters = {\n",
+    "    \"scheduled_time\": \"@trigger().ScheduledTime\",\n",
+    "    \"start_time\": \"@trigger().StartTime\",\n",
+    "    \"utc_now\": \"@utcNow()\",\n",
+    "    \"yesterday\": \"@formatDateTime(subtractFromTime(utcNow(),1,'Day'),'yyyy-MM-dd')\"\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### create a schedule\n",
+    "- [recurrence](https://docs.microsoft.com/en-us/python/api/azureml-pipeline-core/azureml.pipeline.core.schedule.schedulerecurrence?view=azure-ml-py): Defines the frequency, interval and start time of a pipeline Schedule."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "recurrence = ScheduleRecurrence(frequency=\"Day\", interval=1)\n",
+    "recurring_schedule = Schedule.create(ws, name=\"MyRecurringSchedule\",\n",
+    "                                     description=\"Based on time, pipeline param expression\",\n",
+    "                                     pipeline_id=pub_pipeline_id,\n",
+    "                                     pipeline_parameters=pipeline_parameters,\n",
+    "                                     experiment_name=\"Schedule-pipeline-parameter-sample\",\n",
+    "                                     recurrence=recurrence\n",
+    "                                     )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Update a schedule"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Submit the Pipeline every 2 weeks on Monday and Wednesday at 6:30pm UTC\n",
+    "update_recurrence = ScheduleRecurrence(frequency=\"Week\", interval=2, week_days=[\"Monday\", \"Wednesday\"], time_of_day=\"18:30\")\n",
+    "update_pipeline_parameters = {\n",
+    "    \"start_time\": \"@addMinutes(trigger().ScheduledTime, 10)\",\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "recurring_schedule.update(name=\"NewRecurringSchedule\",\n",
+    "                description=\"Based on time, pipeline param expression\",\n",
+    "                pipeline_parameters=update_pipeline_parameters,\n",
+    "                recurrence=update_recurrence)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create a schedule for pipeline endpoint"
```

Comment:
We need to provide an example telling user how to use pipeline parameter expression in create_for_pipeline_endpoint.
Current example looks nothing to do with expression.

There is also one import thing user need to know in schedule pipeline endpoint:
if no pl param expression, just default value, user could update pipeline by just republishing, without Schedule.update.
But for pl param expression, user need to update schedule manually, and republish pipeline will not work.